### ISSUE TITLE
MabiPack Won't Start

### ISSUE THREAD

**Issue by maybecryptic** [logue/MabiPack#4]
![](https://i.imgur.com/MO5ojmc.png)

http://mabipatchinfo.nexon.net/patch/patch.txt no longer works

**Comment by logue** [comment]
xCryptic,
Can you send me Mabinogi.exe?

**Comment by maybecryptic** [comment]
sure

[Mabinogi.zip](https://github.com/logue/MabiPack/files/3259266/Mabinogi.zip)

**Comment by shaggyze** [comment]
mabinogi.exe is outdated and is no longer used, they changed their patching system and now it uses nexon or steam launcher and passes a token to launch client.exe
https://github.com/Hyddwn/HyddwnLauncher/blob/dev/HyddwnLauncher/Network/NexonApi.cs#L88

**Comment by logue** [comment]
In this program, I use it for the reason of getting server version to the last. At this time, patch.txt of the patch server is loaded.
Currently, only the NA client and the Taiwan client can not get the patch version and it is an error.

As it is unlikely that patch server specifications will change depending on the country, it probably don't need to api to get patch.txt.

In any case, it is not a function that is particularly necessary for the purpose of this tool, so I'm thinking about removing it.

**Comment by shaggyze** [comment]
i agree, i mainly use the command-line features of mabipacker which is unaffected by this error, but for a simple solution for everyone else it affects, it would be easier just to remove it.

**Comment by logue** [comment]
Thanks,
For the time being, I have fixed the relevant part.

**Comment by shaggyze** [comment]
verifying that v.1.3.4.2 has solved the issue for me.

**Comment by logue** [comment]
The source code is dirty, so I am recreating it using MahApps.Metro on the WPF branch.
I implemented the command line for a while, but I still have bugs.
